browser/src/App.js

```diff
@@ -215,10 +211,17 @@ function App() {
         <h3>Fees Charged by agency</h3>
         <div className="graph graph--bar">
           <ResponsiveBar
-            isInteractive={false}
-            margin={{ top: 0, right: 100, bottom: 50, left: 100 }}
+            isInteractive
+            margin={{ top: 0, right: 0, bottom: 50, left: 15 }}
             layout="horizontal"
-            enableLabel={false}
+            enableLabel
+            tooltip={config => new Intl.NumberFormat('en-US',{style: 'currency', currency: 'USD'}).format(config.value)}
+            labelFormat={(num) => {
+              const numString = Math.round(num) + '';
```

Comment:
Maybe makes sense to break this into it's own function? I feel like creating currency labels is something we might do a lot of